style.css

```diff
@@ -1,818 +1,682 @@
+
 * {
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-  -moz-box-sizing: border-box;
-  -webkit-box-sizing: border-box;
+  font-family: "Lato", Times, sans-serif;
+  text-decoration: none;
+  list-style-type: none;
 }
 
 body {
-  background-image: url(images/image_geometry_1.png);
+  margin: 0;
+  font-family: 'Lato', Times, sans-serif;
+}
+
+.background {
+  position: absolute;
+  top: 0;
+  z-index: -2;
+  background-image: url('static/desktop_bg.png');
   background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 600px;
+  padding: 110px;
 }
 
-#toolbar {
-  display: flex;
-  justify-content: flex-end;
-  margin: 16px;
+.background-gradient {
+  background-color: #fff;
+  display: none;
+  opacity: 0.96;
+  position: absolute;
+  top: 0;
+  width: 90vw;
+  height: 600px;
+  z-index: -1;
+  overflow-x: hidden;
 }
 
-.desk,
-.bigmage,
-.form-name-message p {
+/* ---------------NAV-BAR------------ */
+
+#menu-button {
   display: none;
+  z-index: 2;
 }
 
-.name {
-  font-family: 'Playfair Display', serif;
-  font-size: 50px;
-  font-weight: 400;
-  line-height: 100%;
-  width: 200px;
-  padding: 16px;
-  margin: 0;
-  margin-top: 50px;
-  color: #1a2236;
-  animation-duration: 3s;
-  animation-name: slidein;
+.close-button {
+  display: none;
 }
 
-@keyframes slidein {
-  from {
-    margin-left: 100%;
-    width: 300%;
-  }
+#header {
+  position: fixed;
+  width: 100vw;
+  top: 0;
+  z-index: 2;
+}
 
-  to {
-    margin-left: 0%;
-    width: 100%;
-  }
+.header-cont {
+  width: 100vw;
+  background-color: #272a31;
 }
 
-.headline {
-  width: 90%;
-  height: 126px;
-  font-family: 'Poppins', sans-serif;
-  font-size: 14px;
-  line-height: 21px;
-  color: #1a2236;
-  padding: 16px;
-  font-weight: 400;
-  letter-spacing: -0.02em;
-  margin-bottom: 40px;
-}
-
-.textBoton {
-  text-decoration: none;
-  font-family: 'Poppins', sans-serif;
-  font-size: 16px;
-  color: #fff;
-  text-align: center;
+.header-container {
+  width: 90vw;
+  display: flex;
+  flex-direction: row-reverse;
+  gap: 12px;
+  padding-top: 4px;
+  padding-bottom: 4px;
 }
```

Comment:
- [ ] As mentioned previously the top dark navbar needs to be horizontally aligned with the bottom nav links as shown in the Behance design. One way you can achieve this is by giving both the navbars the same width and using the `margin-inline: center` property to center them. 😉
- Behance design:
  ![](https://user-images.githubusercontent.com/20554583/242171614-e627d97c-e901-4fea-a23b-80cf86d5597e.png)
- Your design:
  ![image](https://github.com/Bleu-Yves-Sopoude/Bleu-Yves-Sopoude.github.io/assets/104968236/ec94c141-c2fc-44a9-a15c-375fc87fd80c)
- [ ] As you can see from the original design the logout button should be right above _TC campaign_ text. But in your design they're misaligned.